### PR TITLE
MAR-54 - unseen changes

### PR DESCRIPTION
**Pull request by fericsepi**
MAR-54:
- add last_seen_on field to barrier
- new model BarrierUserHits to track user-barrier view timestamps

### CODE REVIEW

File: api/barriers/models.py [hunk 1]
```diff
@@ -220,6 +216,14 @@ def created_user(self):
     def modified_user(self):
         return self._cleansed_username(self.modified_by)
 
+    @property
+    def last_seen_by(self, user_id):
```
Comment (dankingston) [inline api/barriers/models.py]: Guessing it shouldn't be a property if it takes an argument? Is this used anywhere?

File: api/barriers/models.py [hunk 2]
```diff
@@ -243,6 +247,16 @@ def save(
         )
 
 
+class BarrierUserHits(models.Model):
```
Comment (dankingston) [inline api/barriers/models.py]: Model names should really be singular - could we change it to `BarrierUserHit`?

**Comment by dankingston** [comment]
Also nice to see some of the PEP8 stuff cleaned up!

File: api/barriers/models.py (hunk 1 above)
Comment (fericsepi) [reply api/barriers/models.py]: ohyeah

**Comment by fericsepi** [comment]
resolved conflicts and migrations

**Pull request merged**